symfony/lox24-notifier/7.1/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
indentation must be fixed

Reply:
Fixed.

Fixed 'USERNAME' to 'USER'. In LOX24 user has integer id only, not username.